core/assets/bundles/bundle_fr.properties

```
@@ -828,7 +828,7 @@ sector.origin.name = Origin
 
 sector.onset.description = Le secteur de tutoriel. Cet objectif n'a pas encore été créé. Attendez pour des informations futures.
 sector.aegis.description = L'ennemi est protégé par des boucliers. Un module expérimental de briseur de bouclier a été détecté dans ce secteur.\nLocalisez cette structure, approvisionnez-la avec des munitions de tungstène et détruisez la base ennemie.
-sector.lake.description = Le lac de scories du secteur limite les unités utilisables. Les troupes volantes sont la seule option.\nRecherchez le [accent]fabricateur de vaisseaux[] et produisez un [accent]elude[] aussi tôt que possible.
+sector.lake.description = Le lac de scories du secteur limite les unités utilisables. Les troupes volantes sont la seule option.\nRecherchez le [accent]fabricateur de vaisseaux[] et produisez un [accent]Elude[] aussi tôt que possible.
```

```suggestion
sector.lake.description = Le lac de scories du secteur limite les unités utilisables. Les troupes volantes sont la seule option.\nRecherchez le [accent]fabricateur de vaisseaux[] et produisez un [accent]elude[] aussi tôt que possible.
```
"elude" est déjà accentué par la couleur, pas besoin de majuscule.
https://github.com/phinner/mindustry/blob/e4253bcbf8fdc638cc52247ac553b5ed697447d4/core/assets/bundles/bundle.properties#L831